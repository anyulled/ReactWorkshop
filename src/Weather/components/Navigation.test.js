import React from "react";
import Navigation from "./Navigation";
import {shallow} from "enzyme";

describe("Navigation", () => {
    it("renders properly", () => {
        const wrapper = shallow(<Navigation/>);
        expect(wrapper).toMatchSnapshot();
    });
});