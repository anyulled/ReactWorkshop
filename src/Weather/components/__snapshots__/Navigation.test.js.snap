// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navigation />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "bsStyle": "default",
      "children": Array [
        <NavbarHeader>
          <NavbarBrand>
            <a
              href="#"
            >
              Weather App
            </a>
          </NavbarBrand>
        </NavbarHeader>,
        <Nav
          bsClass="nav"
          justified={false}
          pullLeft={false}
          pullRight={false}
          stacked={false}
        >
          <NavItem
            active={false}
            disabled={false}
            eventKey={1}
            href="/"
          >
            App
          </NavItem>
          <NavItem
            active={false}
            disabled={false}
            eventKey={2}
            href="/styled"
          >
            Styled Component
          </NavItem>
        </Nav>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "children": <NavbarBrand>
            <a
              href="#"
            >
              Weather App
            </a>
          </NavbarBrand>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <a
              href="#"
            >
              Weather App
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "Weather App",
              "href": "#",
            },
            "ref": null,
            "rendered": "Weather App",
            "type": "a",
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "bsClass": "nav",
          "children": Array [
            <NavItem
              active={false}
              disabled={false}
              eventKey={1}
              href="/"
            >
              App
            </NavItem>,
            <NavItem
              active={false}
              disabled={false}
              eventKey={2}
              href="/styled"
            >
              Styled Component
            </NavItem>,
          ],
          "justified": false,
          "pullLeft": false,
          "pullRight": false,
          "stacked": false,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "active": false,
              "children": "App",
              "disabled": false,
              "eventKey": 1,
              "href": "/",
            },
            "ref": null,
            "rendered": "App",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "active": false,
              "children": "Styled Component",
              "disabled": false,
              "eventKey": 2,
              "href": "/styled",
            },
            "ref": null,
            "rendered": "Styled Component",
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "bsStyle": "default",
        "children": Array [
          <NavbarHeader>
            <NavbarBrand>
              <a
                href="#"
              >
                Weather App
              </a>
            </NavbarBrand>
          </NavbarHeader>,
          <Nav
            bsClass="nav"
            justified={false}
            pullLeft={false}
            pullRight={false}
            stacked={false}
          >
            <NavItem
              active={false}
              disabled={false}
              eventKey={1}
              href="/"
            >
              App
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={2}
              href="/styled"
            >
              Styled Component
            </NavItem>
          </Nav>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <NavbarBrand>
              <a
                href="#"
              >
                Weather App
              </a>
            </NavbarBrand>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <a
                href="#"
              >
                Weather App
              </a>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Weather App",
                "href": "#",
              },
              "ref": null,
              "rendered": "Weather App",
              "type": "a",
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "bsClass": "nav",
            "children": Array [
              <NavItem
                active={false}
                disabled={false}
                eventKey={1}
                href="/"
              >
                App
              </NavItem>,
              <NavItem
                active={false}
                disabled={false}
                eventKey={2}
                href="/styled"
              >
                Styled Component
              </NavItem>,
            ],
            "justified": false,
            "pullLeft": false,
            "pullRight": false,
            "stacked": false,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "class",
              "props": Object {
                "active": false,
                "children": "App",
                "disabled": false,
                "eventKey": 1,
                "href": "/",
              },
              "ref": null,
              "rendered": "App",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "class",
              "props": Object {
                "active": false,
                "children": "Styled Component",
                "disabled": false,
                "eventKey": 2,
                "href": "/styled",
              },
              "ref": null,
              "rendered": "Styled Component",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
